web/frontend/src/components/movement-sensor/index.svelte

```diff
@@ -35,6 +37,14 @@ const refresh = async () => {
     return;
   }
 
+  const result = await Promise.all([
+    getProperties($robotClient, name),
+    getAccuracy($robotClient, name),
+  ]);
+
+  properties = result[0];
+  accuracy = result[1];
+
   properties = await getProperties($robotClient, name);
```

Comment:
Why are you calling `getProperties` twice now? Do we need to await all of the promises at once, or can we just do:

```suggestion
  properties = await getProperties($robotClient, name);
  accuracy = await getAccuracy($robotClient, name);
```

Reply:
I'd recommend removing this call, and keeping `Promise.all` since that parallelizes the requests.

Reply:
@micheal-parks just to clarify, did you mean removing 
```
  properties = await getProperties($robotClient, name);
  accuracy = await getAccuracy($robotClient, name);
```

Reply:
Should we poll for accuracy? Or is that static?

Reply:
@ethanlook  excellent point! we should poll for accuracy. Can you point me to an example where it's being done?

Reply:
Here's an [example](https://github.com/viamrobotics/rdk/blob/main/web/frontend/src/components/navigation/hooks/use-paths.ts#L14-L29) of polling for nav paths. I think it would look something like this for you:

```
let accuracy: movementsensorApi.GetAccuracyResponse.AsObject | undefined;

useConnect(() => {
  const clearInterval = setAsyncInterval(() => {
    void getAccuracy($robotClient, name);
  }, 1000); // or whatever millisecond interval you want

  return clearInterval;
});
```

Reply:
We're already polling for accuracy here. I wouldn't separate into two different useConnect hooks, because then each request would block the other instead of the parallelism that exists in this PR. I just think line 48 should be removed.

Reply:
🤦🏼  I missed that this was in a refresh loop. Micheal is absolutely correct, apologies.